doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -115,26 +115,26 @@ Networks nodes are mainly used for L3 traffic management for overlay tenant netw
 
 #### 4.2.2.5. Compute Nodes
 
-This section specifies the compute node configurations to support the Basic and High Performance profiles; in OpenStack this would be accomplished by creating "flavors". In certain cases, different nodes (hosts, physical servers) in the OpenStack cloud infrastructure may be configured with a profile but with a different geometry (such as CPU over-suscription). The cloud operator may choose to create a "flavor series" as a set of standard configurations, of a given profile, capturing some of this variability.
+This section specifies the compute node configurations to support the Basic and High Performance profiles; in OpenStack this would be accomplished by specifying the configurations when creating "flavors". The cloud operator may choose to implement certail profile-extensions ([RM 2.4 Profile Extensions](../../../ref_model/chapters/chapter02.md#242-profile-extensions-specialisations)) as a set of standard configurations, of a given profile, capturing some of the variability through different values or extra specifications.
 
 -	The software and hardware configurations are as specified in the [Reference Model chapter 5.4](../../../ref_model/chapters/chapter05.md#5.4)
 -	BIOS requirement
     -	The general BIOS requirements are described in the [Reference Model chapter 5.4](../../../ref_model/chapters/chapter05.md#5.4)
 
 
-**Flavor Series**
+**Example Profile Extensions**
```

Comment:
As the table gathers capabilities related to profile and to extensions, the title can be:
Examples-Profiles and their Extensions

Reply:
@karinesevilla Done - Thanks